status/views.py

```diff
@@ -155,3 +158,16 @@ def view_code(request, sid):
     except IOError:
         logger.warning('File %s Not Found!' % filename)
         raise Http404('File %s Not Found!' % filename)
+
+
+@login_required()
+def rejudge(request, sid):
+    submission = get_object_or_404(Submission, id=sid)
+    if can_rejudge(submission, request.user):
+        rejudge_submission(submission)
+        print request.GET
+        messages.success(request, 'Submission %s rejuded' % sid)
+        return redirect('%s?%s' % \
+            (reverse('status:status'), urllib.urlencode(request.GET)))
```

Comment:
Make status remain in the same page.
We use get parameter  ``?username=&pid=&cid=&status=`` to do that job.